tests/helperServer.js

```diff
@@ -138,6 +142,25 @@ module.exports = {
               callback(null);
             });
           },
+          function (callback) {
+            externalGrader.init(assessment, function (err) {
+              if (ERR(err, callback)) return;
+              callback(null);
+            });
+          },
+          function (callback) {
+            if (!config.externalGradingEnableResults) return callback(null);
+            externalGraderResults.init((err) => {
+              if (ERR(err, callback)) return;
+              callback(null);
+            });
+          },
```

Comment:
This is actually AWS-only code that polls SQS for grading messages. We shouldn't need to do this for tests.

```suggestion
```